docs/.vitepress/data/gen-files.ts

```
@@ -100,6 +77,8 @@ async function fetchRepo(meta: {
       }),
     })
 
+    console.log(results)
```

ESLint

[error] Unexpected console statement. Only these console methods are allowed: warn, error.

(no-console)